minard/templates/burst_l3.html

```diff
@@ -0,0 +1,130 @@
+{% extends "layout.html" %}
+{% block title %}Burst Trigger L3{% endblock %}
+{% block head %}
+  {{ super() }}
+{% endblock %}
+{% block body %}
+  {{ super() }}
+
+    <div class="container">
+      <div class="row">
+        <div class="col-md-12">
+            <h2>Supernova Burst Trigger - Level 3</h2>
+
+            <div class="row">
+                <div class="col-md-12">
+                  <div class="panel panel-default">
+                      <div class="panel-heading">
+                          Search:
+                      </div>
+                      <div class="panel-body">
+                          <form class="form-horizontal" role="form" method="get" action="{{ url_for("burst_l3") }}" onsubmit="return validate_form()">
+                              <div class="form-group form-group-sm">
+                                  <label class="col-sm-2 control-label">
+                                      Search by:
+                                  </label>
+                                  <div class="col-sm-2">
+                                      <select id="search-by" name="search" class="form-control" onchange="update_search()">
+                                          <option value="run">Run</option>
+                                          <option value="gtid">GTID</option>
```

Comment:
I couldn't get results from the search by GTID working. I tried a range of 0 to 10000000 and it returned 0 results.

Reply:
Ah. The search works slightly unconventionally: the range that you input to search needs to be fully covered by the run. So if a run has a GTID range of 1-100, you would need to search for anything between 1-100, ie 5-10 would work but 1-101 would not.

Reply:
The reason behind this is that we usually want to search for a really small range of GTIDs (or even single event). These can come from the noel server breakdowns and similar.